From 522d0471b0e1b845de28e5b577e9d71e633c40bf Mon Sep 17 00:00:00 2001
From: Steve Lhomme <robux4@videolabs.io>
Date: Wed, 11 Jan 2017 15:04:59 +0100
Subject: [PATCH 14/39] dxgi_fmt: log RGB16 floats

Apparently some GPUs handle this format: 16 bits per colour with alpha in float.
---
 modules/video_chroma/dxgi_fmt.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/modules/video_chroma/dxgi_fmt.c b/modules/video_chroma/dxgi_fmt.c
index 67ffbf3..d956085 100644
--- a/modules/video_chroma/dxgi_fmt.c
+++ b/modules/video_chroma/dxgi_fmt.c
@@ -54,6 +54,7 @@ static const dxgi_format_t dxgi_formats[] = {
     { "Y410",        DXGI_FORMAT_Y410,                0                  },
     { "NV11",        DXGI_FORMAT_NV11,                0                  },
     { "RGB10A2",     DXGI_FORMAT_R10G10B10A2_UNORM,   0                  },
+    { "RGB16_FLOAT", DXGI_FORMAT_R16G16B16A16_FLOAT,  0                  },
     { "UNKNOWN",     DXGI_FORMAT_UNKNOWN,             0                  },
 
     { NULL, 0, 0}
-- 
2.10.1.windows.1

